client/pages/_app.tsx

```diff
@@ -17,35 +17,33 @@ import {
   gatewayAddress,
   faucetUri,
   nodeUri,
-  secretStoreUri,
   verbose,
   graphUrl,
   artifactsFolder,
-  correctNetworkId
 } from 'src/config'
-import chainConfig from 'src/chainConfig'
 import { ToastContainer } from 'react-toastify'
 import 'react-toastify/dist/ReactToastify.css'
 import '../src/components/toast/toast.scss'
 
-const appConfig = {
+const appConfig: Config = {
   web3Provider:
     typeof window !== 'undefined'
       ? window?.ethereum
       : new ethers.providers.JsonRpcProvider(nodeUri),
-  nodeUri,
+  web3ProviderUri: nodeUri,
   marketplaceUri,
-  gatewayUri,
+  neverminedNodeUri: gatewayUri,
   faucetUri,
-  gatewayAddress,
-  secretStoreUri,
+  newGateway: true,
+  neverminedNodeAddress: gatewayAddress,
```

Comment:
this should be called the same than the config prop `neverminedNodeAddress`

Reply:
Fixed